pkg/controllers/user/monitoring/projectHandler.go

```diff
@@ -208,6 +208,15 @@ func (ph *projectHandler) deployApp(appName, appTargetNamespace string, appProje
 		appAnswers[mustKey] = mustVal
 	}
 
+	// complete sync target & path
+	if appAnswers["prometheus.sync.mode"] == "federate" {
```

Comment:
Nit: I feel like we can put some repeatable values like `prometheus.sync.target` in constants.

Reply:
Got